backend/index.js

```diff
@@ -0,0 +1,73 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+const bodyParser = require("body-parser");
+const fs = require('fs');
+const cors = require('cors')
+
+app.use(cors())
+
+app.use(bodyParser.urlencoded({ extended: false }))
```

Comment:
bodyParser is used twice, with "extended" true and "extended" false. 
I think we should keep only one, with extended:true